backend/Squirrel.AzureBlobStorage.WebApi/Program.cs

```diff
@@ -0,0 +1,42 @@
+using Squirrel.AzureBlobStorage.WebApi.Extensions;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Configuration
+    .SetBasePath(builder.Environment.ContentRootPath)
+    .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+    .AddJsonFile($"appsettings.{builder.Environment.EnvironmentName}.json", reloadOnChange: true, optional: true)
+    .AddEnvironmentVariables()
+    .Build();
+// Add services to the container.
+builder.Services.RegisterCustomServices();
+builder.Services.AddAzureBlobStorage(builder.Configuration);
+builder.Services.AddControllers();
+builder.Services.AddHealthChecks();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();
+}
+
+app.UseRouting();
+
+app.UseAuthentication();
+app.UseAuthorization();
+
+app.UseEndpoints(endpoints =>
+{
+    endpoints.MapControllers();
+    endpoints.MapHealthChecks("/health");
+});
+
+app.UseHttpsRedirection();
+
+app.Run();
```

Comment:
does it work fine via Swagger without `app.MapControllers();`?

Reply:
+ and Core also works fine without any changes if it starts via IIS Express or docker